### ISSUE TITLE
🏁 SHINKENDO ↔ v18.10.1

### ISSUE THREAD

**Issue by Trent-0** [itchio/itch-compatibility-watchlist#212]
:rotating_light: game [SHINKENDO](https://r-nikaido.itch.io/shinkendo) is broken for me.

:book: Here's the complete [debug log](https://gist.github.com/0569213bce929d23d6c52c5b68246736).

:running: The itch app appears to download the game and acts as though it's been downloaded, but there are no game files in \itch\apps\SHINKENDO (the .itch folder is still in there though). Doing a quick search, it doesn't look like the game's .zip was downloaded at all. Downloading the .zip off the game's page using Firefox works fine.

**Comment by fasterthanlime** [comment]
Is that with butler v1.0.1? (See my response to your previous issue
reports!)

On Sat, Oct 22, 2016, 22:22 Secret-Tunnel notifications@github.com wrote:

🚨 game SHINKENDO https://r-nikaido.itch.io/shinkendo is broken for me.

📖 Here's the complete debug log
https://gist.github.com/0569213bce929d23d6c52c5b68246736.

🏃 The itch app appears to download the game and acts as though it's been
downloaded, but there are no game files in \itch\apps\SHINKENDO (the .itch
folder is still in there though). Doing a quick search, it doesn't look
like the game's .zip was downloaded at all. Downloading the .zip off the
game's page using Firefox works fine.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
https://github.com/itchio/itch-compatibility-watchlist/issues/212, or mute
the thread

**Comment by Trent-0** [comment]
Yup, my itch is up to date! I tried reinstalling the game again today and now it works; not sure what was different a few days ago.

**Comment by fasterthanlime** [comment]
Must've been using the older version of butler (it updates independently
from the app, on app startup) :)
